_docs/tasks/security/per-service-mtls.md

```diff
@@ -9,7 +9,7 @@ type: markdown
 ---
 {% include home.html %}
 
-> If you are using Istio 0.7 or later, please refer to [authentication policy task]({{home}}/docs/tasks/security/authn-policy.html) for alternative (recommended) solution using authentication policy.
+> This page will be deprecated soon, if you are using Istio 0.7 or later, please refer to [authentication policy task]({{home}}/docs/tasks/security/authn-policy.html) for alternative (recommended) solution using authentication policy.
```

Comment:
Should we say the feature will be deprecated, instead of talking about the page? e.g

This feature will soon be deprecated. If you are using Istio 0.7 or later, please refer to [authentication policy]({{home}}/docs/tasks/security/authn-policy.html) for the recommended approach moving forward.

Reply:
fixed, thanks!